lib/cli/file-set.js

```diff
@@ -68,8 +68,20 @@ function one(fileSet) {
 function FileSet(cli) {
     var self = this;
 
+    /**
+     * Access files in the file-set.
+     *
+     * @member {Array.<VFile>}
+     */
     self.contents = [];
+
+    /**
+     * Track the file-paths of files when they were added.
+     *
+     * @member {Array.<string>}
+     */
     self.sourcePaths = [];
```

Comment:
We certainly can, I extracted them from your changes, which basically explained some confusing parts. Instead of having those descriptions all over the place, I instead opted for adding (rewritten) commits to the place where they are added.

I’m fine with both cases.